packages/fxa-auth-server/test/local/senders/emails/fluent-localizer.ts

```diff
@@ -2,13 +2,155 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+import { FluentBundle, FluentResource } from '@fluent/bundle';
 import chai, { assert } from 'chai';
 import chaiAsPromised from 'chai-as-promised';
-import { splitPlainTextLine } from '../../../../lib/senders/emails/fluent-localizer';
+import FluentLocalizer, {
+  splitPlainTextLine,
+  parseAcceptLanguage,
+} from '../../../../lib/senders/emails/fluent-localizer';
+
+import { NodeLocalizerBindings } from '../../../../lib/senders/emails/localizer-bindings-node';
 
 chai.use(chaiAsPromised);
 
 describe('fluent localizer', () => {
+  describe('fetches bundles', () => {
+    let LocalizerBindings = new NodeLocalizerBindings();
+    let localizer = new FluentLocalizer(LocalizerBindings);
+
+    it('fails with a bad localizer 1l0n basePath', () => {
+      assert.throws(() => {
+        let LocalizerBindings = new NodeLocalizerBindings({
+          l10n: {
+            basePath: '/not/a/apth',
+          },
+        });
+        new FluentLocalizer(LocalizerBindings);
+      }, 'Invalid l10n basePath');
+    });
+
+    it('produces the current locales', async () => {
+      const { currentLocales } = await localizer.setupLocalizer(
+        'de-CH,it;q=0.8,en-US;q=0.5,en;q=0.3'
+      );
+
+      assert.deepEqual(currentLocales, ['de', 'it', 'en-US', 'en']);
+    });
+
+    it('selects the proper locale', async () => {
+      const { selectedLocale } = await localizer.setupLocalizer(
+        'de-DE,en-US;q=0.7,en;q=0.3'
+      );
+
+      // This needs investigation.
```

Comment:
I'm taking out this comment since I don't think this is true any longer.

Also fixing a typo in a comment and adding another comment.